unpack_action.go

```diff
@@ -51,12 +51,7 @@ func (pf *UnpackAction) Verify(context *DebosContext) error {
 
 func (pf *UnpackAction) Run(context *DebosContext) error {
 	pf.LogStart()
-
-	//Trying to get a filename from exports first
-	infile, found := context.origins[pf.File]
-	if !found {
-		infile = path.Join(context.artifactdir, pf.File)
-	}
+	infile := path.Join(context.artifactdir, pf.File)
```

Comment:
This should be in a seperate patch

Reply:
Well actually tin the patch that introduced the issue

Reply:
done